### ISSUE TITLE
Status Bar cannot show white color properly

### ISSUE THREAD

**Issue by allworldg** [tmux/tmux#3291]
### Issue description
Hi, I try to set my status bar backgound with white color .  But it shows grey color in the screen .
![image](https://user-images.githubusercontent.com/48782874/183249895-beccc20c-dcf0-4546-a748-80aaa08adefd.png)

my setup:
```
set -g default-terminal "xterm-256color"
set -g status-style bg=white,fg=orange
```
### my information
* tmux version : tmux 3.3a.
* Platform : Linux x86_64 , Ubuntu20.02 Lts in WSL2 with Windows Terminal .
* $TERM inside: xterm-256color, outside of tmux: xterm-256color.
* Logs :
[tmux-client-4575.log](https://github.com/tmux/tmux/files/9275101/tmux-client-4575.log)
[tmux-out-4577.log](https://github.com/tmux/tmux/files/9275102/tmux-out-4577.log)
[tmux-server-4577.log](https://github.com/tmux/tmux/files/9275103/tmux-server-4577.log)

**Comment by delameter** [comment]
TL;DR
use `bg=brightwhite`

**Comment by allworldg** [comment]
@delameter Thank you. However,It is still grey .  Maybe it is my windows terminal problem . So if guys cannot solve it , just close it. Not big problem .

![image](https://user-images.githubusercontent.com/48782874/183562382-e23701a5-0833-4fa8-a836-e01480aed7c5.png)

**Comment by delameter** [comment]
You can use RGB hex format if your terminal supports it.

`bg=#fffffff`

**Comment by allworldg** [comment]
@delameter It works ! Thank you .
